test/CMakeLists.txt

```diff
@@ -107,6 +107,7 @@ set(TILEDB_UNIT_TEST_SOURCES
   src/helpers.h
   src/helpers.cc
   src/helpers-dimension.h
+  src/serialization_wrappers.cc
```

Comment:
Shouldn't this be in `TILEDB_TEST_SUPPORT_SOURCES` above? Unclear why we duplicate the helpers here.